eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/JsonUtils.java

```diff
@@ -62,72 +63,72 @@ public static String serialize(Object obj) {
         }
     }
 
-    public static <T> byte[] serialize(String topic, Class<T> data) throws JsonProcessingException {
-        if (Objects.isNull(data)) {
+    public static byte[] toBytes(Object obj) {
```

Comment:
Suggest use toJsonBytes is better.

Reply:
@jonyangx  I will fix it and resubmit later